arangod/Aql/SharedQueryState.cpp

```diff
@@ -97,7 +97,7 @@ void SharedQueryState::queueHandler() {
     return;
   }
       
-  bool queued = scheduler->queue(RequestLane::CLIENT_AQL,
+  bool queued = scheduler->queue(RequestLane::CLUSTER_AQL,
```

Comment:
why is low priority desirable?

Reply:
`CLUSTER_AQL` goes to MEDIUM priority. Before that, we had `CLIENT_AQL`, which was LOW and that was bad, because in case of a long queue of actual customer requests this lead to a pile of finished queries not being finalised. The idea here was to put it to medium which is the same priority as cluster internal AQL traffic. I was thinking about something on HIGH, but thought that this might then prevent actual HIGH prio traffic from getting through in an overload scenario.